File: src/main/java/net/minestom/server/command/builder/suggestion/SuggestionCallback.java
package net.minestom.server.command.builder.suggestion;

import net.minestom.server.command.CommandSender;
import net.minestom.server.command.builder.CommandContext;

@FunctionalInterface
public interface SuggestionCallback {
    void apply(CommandSender sender, CommandContext context, Suggestion suggestion);
}

File: src/main/java/net/minestom/server/command/builder/suggestion/SuggestionCallback.kt
package net.minestom.server.command.builder.suggestion

import net.minestom.server.command.CommandSender
import net.minestom.server.command.builder.suggestion.SuggestionEntry
import net.minestom.server.command.builder.CommandContext

fun interface SuggestionCallback {
    fun apply(sender: CommandSender, context: CommandContext, suggestion: Suggestion)
}